config/triplea/schemas/triplea_maps.json

```diff
@@ -0,0 +1,46 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Maps set",
+  "description": "A set of maps",
+  "type": "array",
+  "items": {
+    "title": "Map",
+    "description": "A map",
+    "type": "object",
+    "properties": {
+      "mapName": {
+        "description": "The map name",
+        "type": "string"
+      },
+      "mapCategory": {
+        "description": "The map category",
+        "type": "string",
+        "enum": ["BEST", "GOOD", "DEVELOPMENT", "EXPERIMENTAL"]
+      },
+      "mapType": {
```

Comment:
This property is optional simply because the vast majority of entries in `triplea_maps.yaml` don't define it.  The code assumes `MAP` if it is not defined.  Should I require it and update `triplea_maps.yaml` as needed?